install_files/ansible-base/roles/app/tasks/setup_cron.yml

```diff
@@ -20,6 +20,6 @@
   cron:
     name: Update the number of submissions in the past 24h
     job: "{{ securedrop_code }}/manage.py were-there-submissions-today"
-    special_time: daily
+    hour: "{{ daily_reboot_time - 1 }}"
```

Comment:
This needs to have a ` | default(4)` because prod playbooks won't have that value in them, so current admins will have things break.

Reply:
Also. Why -1?

Reply:
> This needs to have a `| default(4)` because prod playbooks won't have that value in them, so current admins will have things break.

For other reviewers, this statement was false. There is a default in a vars file in ansible, so adding a `default` filter is unnecessary.